maria-bucharest.md

```diff
@@ -0,0 +1,17 @@
+# Bucharest
+
+### Places to eat
+ - Jourmey Pub
+ - Van Gogh
+ - Eden
```

Comment:
I've never been to Bucharest but I know it has a very rich and old history.  Could you be sure to add in those @maria ?